charmtools/build/builder.py

```diff
@@ -1131,6 +1140,8 @@ def main(args=None):
                         help='Same as --log-level=DEBUG')
     parser.add_argument('-c', '--force-color', action="store_true",
                         help="Force raw output (color)")
+    parser.add_argument('--charm-file', '-F', action="store_true",
+                        help="Create a .charm file in the current directory")
```

Comment:
Seems a bit dangerous to drop this in the current directory, e.g. building the easyrsa charm:

```
$ charm-build -F

$ du -h easyrsa.charm
5.1M    easyrsa.charm

$ charm-build -F

$ du -h easyrsa.charm
10M     easyrsa.charm
```

Maybe CHARM_BUILD_DIR would be a better place to put the .charm file?

Reply:
It would be, but `charmcraft` doesn't actually allow you to specify where the file will end up and I wanted to match the behavior of that tool exactly for consistency.

Reply:
I see. Would it make sense to have the -F flag delete the .charm file prior to build, if it already exists?

Reply:
Added to `DEFAULT_IGNORES` so that we avoid including the `.charm` file in itself.  `charmcraft` needs to do the same.

Reply:
Sounds good to me. Thanks!